warehouse/templates/pages/help.html

```diff
@@ -116,6 +118,12 @@ <h3 id="packages">{{ packages() }}</h3>
           A <em>file</em>, also known as a <em>package</em>, on PyPI is something that you can download and install. Because of different hardware, operating systems, and file formats, a release may have several files (packages), like an archive containing source code or a binary <a href="https://pypi.org/project/wheel/#files">wheel</a>.
         </p>
 
+        <h3 id="trove-classifier">{{ trove_classifier() }}</h3>
+
+        <p>Each project's maintainers provide PyPI with a list of "trove classifiers" to categorize each release, describing who it's for, what systems it can run on, and how mature it is.</p>
+        <p>These classifiers can then be used by community members to find projects based on their desired criteria.</p>
```

Comment:
I showed this wording to another maintainer (my spouse Leonard, thanks Leonard!) and he suggested:

> These standardized classifiers can then be used

so let's do that to get across that they're standardized across the community.